video/out/gpu/hwdec.c

```diff
@@ -49,9 +49,11 @@ const struct ra_hwdec_driver *const ra_hwdec_drivers[] = {
     &ra_hwdec_videotoolbox,
 #endif
 #if HAVE_D3D_HWACCEL
+ #if HAVE_EGL_ANGLE
     &ra_hwdec_d3d11egl,
     &ra_hwdec_d3d11eglrgb,
- #if HAVE_D3D9_HWACCEL
+ #endif
+ #if HAVE_D3D9_HWACCEL && HAVE_EGL_ANGLE_WIN32
```

Comment:
should probably be nested for consistency with HAVE_D3D11, HAVE_D3D9_HWACCEL

Reply:
`HAVE_D3D11` is used for build option `d3d11`, meaning 'Direct3D 11 video output', seeming not necessary for `ra_hwdec_dxva2egl` .